build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -60,15 +62,25 @@
                             SetupRestoreTestsInputs;"
           Inputs="@(RestoreTestsInputs)"
           Outputs="@(RestoreTestsInputs->'%(RelativeDir)project.lock.json')">
-    <Exec Command="$(DotnetUnderTest) restore $(TestDirectory) --fallbacksource $(TestPackagesDir)" WorkingDirectory="$(TestDirectory)" />
+    <DotNetRestore  FallbackSource="$(TestPackagesDir)"
+                    ToolPath="$(DotnetUnderTest)" 
+                    WorkingDirectory="$(TestDirectory)" />
   </Target>
  
   <Target Name="CleanTestProjectsBinObj" 
           DependsOnTargets="SetupRestoreTestsInputs;" 
           Inputs="@(RestoreTestsInputs)"
           Outputs="@(RestoreTestsInputs->'%(RelativeDir)project.lock.json')">
-    <Delete Files="$(TestDirectory)/**/bin/**/*.*')" />
-    <Delete Files="$(TestDirectory)/**/obj/**/*.*')" />
+    <ItemGroup>
+      <FilesToDelete Include="$(TestDirectory)**/bin/**/*.*" />
+      <FilesToDelete Include="$(TestDirectory)**/obj/**/*.*" />
+    </ItemGroup>  
+
+    <Delete Files="@(FilesToDelete)" />
+
+    <ItemGroup>
+      <FilesToDelete Remove="*" />
```

Comment:
Yes. FilesToDelete is a local that would ideally be discarded but <Delete> does not do wildcard expansion on its own.